internal/resources/fabric/precision_time/resource_schema.go

```diff
@@ -0,0 +1,280 @@
+								Description: "Represents the domain number associated with the PTP profile. This is used to differentiate multiple PTP networks within a single physical network.",
+								Optional:    true,
+								Computed:    true,
+							},
+							"priority_1": schema.Int64Attribute{
+								Description: "Specifies the priority level 1 for the clock. The value helps in determining the best clock in the PTP network. Lower values are considered higher priority.",
+								Optional:    true,
+								Computed:    true,
+							},
+							"priority_2": schema.Int64Attribute{
+								Description: "Specifies the priority level 2 for the clock. It acts as a tie-breaker if multiple clocks have the same priority 1 value. Lower values are considered higher priority.",
+								Optional:    true,
+								Computed:    true,
+							},
+							"log_announce_interval": schema.Int64Attribute{
+								Description: "Represents the log2 interval between consecutive PTP announce messages. For example, a value of 0 implies an interval of 2^0 = 1 second.",
+								Optional:    true,
+								Computed:    true,
+							},
+							"log_sync_interval": schema.Int64Attribute{
+								Description: "Represents the log2 interval between consecutive PTP synchronization messages. A value of 0 implies an interval of 2^0 = 1 second.",
+								Optional:    true,
+								Computed:    true,
+							},
+							"log_delay_req_interval": schema.Int64Attribute{
+								Description: "Represents the log2 interval between consecutive PTP delay request messages. A value of 0 implies an interval of 2^0 = 1 second.",
+								Optional:    true,
+								Computed:    true,
+							},
+							"transport_mode": schema.StringAttribute{
+								Description: "Mode of transport for the Time Precision Service.",
+								Optional:    true,
+								Computed:    true,
+								Validators: []validator.String{
+									stringvalidator.OneOf(
+										string(fabricv4.PTPADVANCECONFIGURATIONTRANSPORTMODE_MULTICAST),
+										string(fabricv4.PTPADVANCECONFIGURATIONTRANSPORTMODE_UNICAST),
+										string(fabricv4.PTPADVANCECONFIGURATIONTRANSPORTMODE_HYBRID),
+									),
+								},
+							},
+							"grant_time": schema.Int64Attribute{
+								Description: "Unicast Grant Time in seconds. For Multicast and Hybrid transport modes, grant time defaults to 300 seconds. For Unicast mode, grant time can be between 30 to 7200.",
+								Optional:    true,
+								Computed:    true,
+							},
+						},
+					},
+				},
+			},
+			"project": schema.SingleNestedBlock{
+				Description: "An object that contains the Equinix Fabric project_id used for linking the Time Precision Service to a specific Equinix Fabric Project",
+				CustomType:  fwtypes.NewObjectTypeOf[ProjectModel](ctx),
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
+				Attributes: map[string]schema.Attribute{
+					"project_id": schema.StringAttribute{
```

Comment:
Nit: it looks like there's only one of these project blocks and it has a single attribute; why not make `project_id` a top-level attribute of this resource?